pb-jelly-gen/codegen/codegen.py

```diff
@@ -1012,7 +1012,9 @@ def gen_msg(
         with block(self, "impl ::pb_jelly::Message for " + name):
             with block(
                 self,
-                "fn descriptor(&self) -> ::std::option::Option<::pb_jelly::MessageDescriptor>",
+                "const DESCRIPTOR: ::std::option::Option<::pb_jelly::MessageDescriptor> = ",
+                start=" (",
```

Comment:
this causes a double space :p
though is there any point to having the parens at all?